app/app.go

```diff
@@ -846,5 +906,48 @@ func (app *App) registerUpgradeHandlers() error {
 		app.SetStoreLoader(upgradetypes.UpgradeStoreLoader(upgradeInfo.Height, &storeUpgrades))
 	}
 
+	app.UpgradeKeeper.SetUpgradeHandler("v2.2.0-alpha1", func(ctx sdk.Context, plan upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
+		// 1st-time running in-store migrations, using 1 as fromVersion to
+		// avoid running InitGenesis.
+		fromVM := map[string]uint64{
+			"auth":         1,
+			"bank":         1,
+			"capability":   1,
+			"crisis":       1,
+			"distribution": 1,
+			"evidence":     1,
+			"gov":          1,
+			"mint":         1,
+			"params":       1,
+			"slashing":     1,
+			"staking":      1,
+			"upgrade":      1,
+			"vesting":      1,
+			"ibc":          1,
+			"genutil":      1,
+			"transfer":     1,
+			"aol":          1,
+			"did":          1,
+			"burn":         1,
+			"wasm":         1,
+			"datadeal":     1,
+			"datapool":     1,
+			"oracle":       1,
+		}
+
+		app.IBCKeeper.ConnectionKeeper.SetParams(ctx, ibcconnectiontypes.DefaultParams())
```

Comment:
For in-place ibc migration, a new param, `MaxExpectedTimePerBlock`, should be set.

(https://github.com/cosmos/ibc-go/blob/main/docs/migrations/sdk-to-v1.md#in-place-store-migrations)